dll/win32/shell32/folders/CDrivesFolder.cpp

```diff
@@ -572,7 +572,15 @@ static const shvheader MyComputerSFHeader[] = {
     {IDS_SHV_COLUMN_COMMENTS, SHCOLSTATE_TYPE_STR, LVCFMT_LEFT, 10},
 };
 
-#define MYCOMPUTERSHELLVIEWCOLUMNS 5
+enum drives_columns
+{
+    DRIVES_COL_NAME,
+    DRIVES_COL_TYPE,
+    DRIVES_COL_TOTAL_SIZE,
+    DRIVES_COL_FREE_SIZE,
+    DRIVES_COL_COMMENTS,
+    DRIVES_COL_COUNT
```

Comment:
Minor suggestion, maybe name these columns after the existing `IDS_SHV_COLUMN_*`, and rename the last one as `DRIVES_COL_MAX`:
```suggestion
    DRIVES_COL_DISK_CAPACITY,
    DRIVES_COL_DISK_AVAILABLE,
    DRIVES_COL_COMMENTS,
    DRIVES_COL_MAX
```

Reply:
Ok I was able to use existing `MyComputerSFHeader` structure, effectively avoiding an additional enum. 🙂

Reply:
Nice find and improvement. Thanks.